### DISCUSSION TITLE
How to debug Avante

### DISCUSSION THREAD

**Discussion by AndersBennedsgaard** [yetone/avante.nvim#754] [Q&A]
I've tried to set up Avante with the Azure provider, and can't seem to get any responses when trying it out. I would like to be able to do my own debugging instead of creating _yet another provider issue_, but I can't seem to find any special logs, metrics, errors, or anything that is related to my problem.
Is this provided somewhere?

I've found `~/.local/state/nvim/avante/<file>.json`, but that does not seem to have any special information

**Comment by aarnphm** [comment]
set `debug=True`

**Reply by aarnphm** [reply]
There is a toggle debug keybind as well

**Reply by AndersBennedsgaard** [reply]
I can't seem to find any changes when setting `debug=true`, such as why it keeps timing out with `Generating Response` (yes, I have credits - I just confirmed access with a manual `curl`). Is there a way to confirm this is working as intended?
